src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -384,11 +384,11 @@ public static class TreeExtensions
         }
 
         /// <summary>
-        /// Predict a target using a decision tree regression model trained with the <see cref="FastForestBinaryTrainer"/>.
+        /// Creates <see cref="FastForestBinaryTrainer"/>, which predicts a target using a decision tree regression model.
```

Comment:
>Creates [](start = 12, length = 7)

Creates -> Create
Terseness guideline: Use the shortest correct word.

Reply:
weird, seems grammatically incorrect but @natk will know better because she is a native speaker...

---
In reply to: [276787637](https://github.com/dotnet/machinelearning/pull/3399#discussion_r276787637) [](ancestors = 276787637)